### ISSUE TITLE
It's very quiet...

### ISSUE THREAD

**Issue by ghost** [ZasEditor/Zas-Editor-Issues#65]
@ZasEditor May I suggest the devs put up a short status notice? I'm not talking about a long blog post, just a "we're working on things". Perhaps the team is small/tiny and busy working on Zas or needed some time off. That's ok, but you've released a commercial product. I get no response when asking about invoice or e-mail attached to the license, i.e. if I lose my licences code how would (could) it even be retrieved? Twitter is also silent since release and the only responses I see here are made around release day.

**Comment by jsumner** [comment]
Agree it would be good to get an update. It's been very quiet after the initial flurry and while there are some possible reasons the silence is a little worrying.

**Comment by apertureless** [comment]
I guess this is just a classic exit scam.

Just look at all the social media profiles. Reddit, HN, Twitter, Github. All created on april. The domain also registered on april. 
Then a few post to create fuzz, cash grab and silence.

No proper invoice after purchase, no proper license key, no email nothing. That is not how you run a legal business.
And no updates since then. Not even the most basic bug fixes like the failing update. 
Shady as fuck.

**Comment by ghost** [comment]
Yes, at this point that's what it looks like. Oddly ambitious, though, since even if they used someone else's code to pull it off it kind of works...? EDIT: The app is also notarized...? Some visual similarities to https://github.com/CodeEditApp/CodeEdit but that may be a coincidence. The $25 price tag is also perhaps low enough for people to take the risk.

The physical contact address points to a spa...

@ZasEditor From this point on I'll treat this as fraud and report the purchase as such with my bank.

**Comment by Agrailag** [comment]
https://www.zeditor.app/ now return 404. So @apertureless tesis looks right about scam.
![2022-05-14_16-31](https://user-images.githubusercontent.com/37513902/168428009-99e55941-798b-4eee-b451-f6e10ca80f88.png)

**Comment by ghost** [comment]
I guess this should be reported to Apple as well, since the dev account should be deactivated or the actual dev whose credentials were used (if that's the case) should be informed.

Wonder whose code was used/stolen, since it was semi-functional. A simple `strings` on the app make it seem it's using xi-editor for the backend.

By the way - just to be extra careful - has anyone detected anything malicious with the app itself, other than being unfinished? Payloads/odd network behaviour or unknown processes etc - those kind of horrors.

**Comment by ack3rs** [comment]
<img width="559" alt="Screenshot 2022-05-16 at 10 44 05" src="https://user-images.githubusercontent.com/3899682/168565562-17871ea0-28e4-4f18-a9f8-546004ccefaa.png">

Only Call it seems to make is to https://zas-editor-stripe-server.herokuapp.com,  not detected anything else, so far at least. 

There is a stripe key in the source on that URL,  This should be reported to. Stripe.

**Comment by avdept** [comment]
I don't get one thing. If they tried to scam ppl, this is very niche tool, so I highly double they'd get any reasonable amount, assuming you'd need to spend a lot of time to build their initial version, including support for language server

**Comment by ghost** [comment]
It's indeed an odd app choice for a hit-and-run money grab. Even if it were the case that the code was stolen it still seems like a lot of work for scamming a relatively small (?) target group.

**Comment by apertureless** [comment]
Well, maybe we're wrong. Maybe it was just an hobby project made in free time. And then tried to monatize it and was overhelmed by legal stuff, feedback and issues. Who knows... but I do not expect any updates. Sadly.

**Comment by austincondiff** [comment]
This is really unfortunate and I hope this is a mistake and not a scam, but it certainly looks bad. If it turns out that it is, feel free to check out our project we've been building – [CodeEdit](https://github.com/CodeEditApp/CodeEdit). It is similar to Zas Editor, but it is free and open source. We have a growing community so if anything goes wrong we are all in this together to make things right!

**Comment by ColourGrey** [comment]
Currently, [their website](https://www.zeditor.app) is unreachable because their security certificate expired on July 2 2022.
